examples/tensorflow/requirements.txt

```diff
@@ -4,4 +4,4 @@ tensorflow_hub
 tensorflow_addons==0.14.0
 opencv-python
 protobuf==3.17.3
-git+https://github.com/alexsu52/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI
+git+https://github.com/andrey-churkin/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI
```

Comment:
Looks like that we need to keep `cocoapi.git` on the public side. I'm skeptical to add a dependency on private repositories.

Reply:
I think it's good. I don't know if there is specific reason to use private one, but there was no problem when I tested on pycocotools==2.0.0, so maybe change it to that version would be good.